src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -540,6 +547,13 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
     noshort = true
   )
 
+  val assumeInjectivityOnInhale: ScallopOption[Boolean] = toggle(
+    name = "assumeInjectivityOnInhale",
+    descrYes = "Assumes injectivity of predicates on inhale, like in Viper versions previous to 2022.02 (default)",
```

Comment:
```suggestion
    descrYes = "Assumes injectivity of the receiver expression when inhaling quantified permissions, instead of checking it, like in Viper versions previous to 2022.02 (default)",
```

Reply:
This is the description of Viper's config option